### ISSUE TITLE
[bug] vitest test function is detected as component

### ISSUE THREAD

**Issue by ulrichstark** [Rel1cx/eslint-react#1527]
### Describe the bug

A vitest test function is detected as component and therefore lint rules like @eslint-react/rules-of-hooks in my reproduction report false positives.

### Reproduction

```ts
import { test, vi } from "vitest";

test("", () => {
  vi.useFakeTimers();
});
```

### Expected behavior

Not being detected as component => no reports

### Platform and versions

```Text
@eslint-react/eslint-plugin: 3.0.0-beta.18
```

### Stack trace

```Text

```
